articles/cosmos-db/how-to-always-encrypted.md

```diff
@@ -72,9 +72,9 @@ For each property that you want to encrypt, the encryption policy defines:
 
 The Azure Cosmos DB service never sees the plain text of properties encrypted with Always Encrypted. However, it still supports some querying capabilities over the encrypted data, depending on the encryption type used for a property. Always Encrypted supports the following two types of encryptions:
 
-- **Deterministic encryption:** It always generates the same encrypted value for any given plain text value and encryption configuration. Using deterministic encryption allows queries to perform equality filters on encrypted properties. However, it may allow attackers to guess information about encrypted values by examining patterns in the encrypted property. This is especially true if there's a small set of possible encrypted values, such as True/False, or North/South/East/West region.
+- **Deterministic encryption:** It always generates the same encrypted value for any given plain text value and encryption configuration. Using deterministic encryption allows queries to perform equality filters on encrypted properties. However, it may allow attackers to guess information about encrypted values by examining patterns in the encrypted property. This is especially true if there's a small set of possible encrypted values, such as True/False, or North/South/East/West region. Deterministc encryption uses the same Initialization Vector (IV) for each encrypted property.
```

Comment:
Technically this isn't correct, the IV isn't the same for each property. IV generation for Always Encrypted is documented [here](https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/always-encrypted-cryptography?view=sql-server-ver15#step-1-generating-the-initialization-vector-iv) and it may be better to just reference that page.

Reply:
agree 100% if there's existing docs...

Reply:
Thanks @x509cert ! The doc has been updated with the corresponding link and will be published later today. You can cancel this PR.